server/database/build.sql

```diff
@@ -0,0 +1,40 @@
+BEGIN;
+
+CREATE TABLE voters(
+  id INT PRIMARY KEY,
+  last_name VARCHAR(20) NOT NULL,
+  first_name VARCHAR(20) NOT NULL,
+  middle_name VARCHAR(20) NOT NULL,
+  box_number SMALLINT NOT NULL,
+  address VARCHAR(100) NULL,
+  serial_box_number INT NOT NULL,
+  telephone INT NULL,
+  status SMALLINT DEFAULT 1,
+  with_against SMALLINT DEFAULT 2
+);
+
+CREATE TABLE region(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50)
+);
+
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(100) UNIQUE NOT NULL,
+  user_name VARCHAR(20) UNIQUE NOT NULL,
+  password VARCHAR(20) NOT NULL,
+  telephone INT NOT NULL,
```

Comment:
Why is telephone set to `Int`? Is that supposed to hold a phone number? If yes, probably better to set it to alphanumerical charset (so that it allows for '-' or '+' signs if necessary). Just a regular `varchar` data type should do.